DOCKER-network/dockernet

```diff
@@ -0,0 +1,264 @@
+import argparse
```

Comment:
Can you add '.py' extension so IDE can apply proper intelligence based on source code type?

Reply:
The reason why I removed the extension was so the command would be easier to type, you can also just manually tell your IDE to associate the file with python

Reply:
Fine.